book/history/sourdough-history.tex

```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
     lessons from the past.
 \end{quoting}
 
-Sourdough has been made since ancient times. The exact origins of fermented
+The story of sourdough bread begins in ancient oceans. These oceans were the
+birthplace of all Earth's life. To understand this timeline, envision Earth's
+history compressed into one year. On this scale, January 1st signifies Earth's
+formation 4.54 billion years ago. Midnight on December 31st is our present.
+Humans appeared on the evening of December 31st. The dinosaurs, once our
+planet's rulers, vanished on December 29th. Earlier on that same day, humans
+made their debut. Just seconds ago, the first firecrackers were lit.
+
+On March 25th, the oceans birthed the first single-celled bacteria. In these
+waters, another single-celled life form, \emph{archaea}, also thrived. These
+organisms inhabit extreme environments, from boiling vents to icy waters.
+
+\begin{figure}[!htb]
+\begin{center}
+  \input{figures/fig-life-planet-sourdough-timeline.tex}
+  \caption[Sourdough microbiology timeline]{Timeline of significant events
+    starting from the first day of Earth's existence,
+    divided into months, and extending to the present day,
+    marked at midnight. This visualization shows the pivotal steps
+    of life and sourdough on earth.}%
+\end{center}
+\end{figure}
+
+The first arrivals, bacteria or archaea, remains debated. For three
+months (or approximately 1.1 billion years), these life forms dominated
+the oceans. Then, on June 25th in an highly unlikely event, an archaeon consumed a bacterium.
+Instead of digesting it, they formed a symbiotic relationship. This led to the
+first nucleated organisms, marking an evolutionary milestone. This event lead
+to the development of plants, fungi and also ultimately humans.
+
+Life stayed aquatic for another three months.
+On October 4th, bacteria first colonized land. By October 15th, the
+first aquatic fungi appeared. They adapted and, by November 24th, had colonized
+land.
+
+By December 3rd, yeasts emerged on land. This laid groundwork for bread-making.
+Jump 140 million years to December 14th, and dinosaurs arose. They reigned
+until December 29th when they faced extinction. Another 25 million
+years later, or our timeline's 2 days, humans appeared.
+
+A few hours later after the arrival of humans, a more subtle culinary
+revolution was unfolding. By 12,000 BC, just 5 seconds before our metaphorical
```

Comment:
Use Siunit package here there is an example further down the text \num{12000}~BC